services/skipchain/api.go

```diff
@@ -0,0 +1,130 @@
+package skipchain
+
+import (
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/cothority/lib/sda"
+)
+
+// Client is a structure to communicate with the Skipchain
+// service from the outside
+type Client struct {
+	*sda.Client
+}
+
+// NewClient instantiates a new client with name 'n'
+func NewClient() *Client {
+	return &Client{Client: sda.NewClient("Skipchain")}
+}
+
+// CreateRootInter creates two Skipchains: a root SkipChain with
+// maximumHeight of maxHRoot and an intermediate SkipChain with
+// maximumHeight of maxHInter. It connects both chains for later
+// reference.
+func (c *Client) CreateRootInter(elRoot, elInter *sda.EntityList, maxHRoot, maxHInter int, ver VerifierID) (root, inter *SkipBlock, err error) {
```

Comment:
Ok after discussion with @ineiti , I get it that a `Propose{Roster,Data}` can be used as `CreateSkipchain` if the block you give does not have any back links.